active.go

```diff
@@ -69,8 +70,8 @@ func (as *ActiveService) FindByGitHubId(ctx context.Context, githubId uint, opt
 	var ar activeResponse
 	resp, err := as.client.Do(ctx, req, &ar)
 	if err != nil {
-		return nil, resp, err
+		return nil, newResponse(resp, ar.Pagination), err
 	}
 
-	return ar.Builds, resp, err
+	return ar.Builds, newResponse(resp, ar.Pagination), err
```

Comment:
Sorry but I'm not a big fun of this adhoc `newResponse` method...

Reply:
Yeah, I copied the logic from the github golang API.

https://github.com/google/go-github/blob/master/github/github.go#L532

But they have their own [client](https://github.com/google/go-github/blob/master/github/github.go#L159) for all the responses, so it's done outside each function.

It's probably better to just change all methods to return the `*Response` instead of some `*http.Response` and some not.